### PR TITLE
updated for derpitcraft

### PR DESCRIPTION
**Pull request by nrbxyz**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,24 +12,24 @@
     <!-- meta for embeds -->
     <meta name="title" content="TrixMC">
     <meta name="description" content="A free Minecraft server network unlike any other. 
-    Supports Minecraft Java and Bedrock 1.7.X to 1.17.X, with cracked client support.
+    Supports Minecraft Java and Bedrock 1.8.X to 1.17.X, with cracked client support.
```
Comment (vachanmn123) [inline index.html:15]: max version to be 1.18.1

File: index.html [hunk 2]
```diff
@@ -12,24 +12,24 @@
...
     
-    Currently offering Bedwars and Skyblock, with more to come.">
+    Currently offering Derpitcraft, Bedwars and Skyblock, with more to come.">
     <meta property="og:type" content="website">
     <meta property="og:url" content="https://trix.tk/">
-    <meta property="og:title" content="The Trix  Network">
+    <meta property="og:title" content="TrixMC">
     <meta property="og:description" content="A free Minecraft server network unlike any other. 
-    Supports Minecraft Java and Bedrock 1.7.X to 1.17.X, with cracked client support.
+    Supports Minecraft Java and Bedrock 1.8.X to 1.17.X, with cracked client support.
```
Comment (vachanmn123) [inline index.html:22]: max version to be 1.18.1

File: index.html [hunk 3]
```diff
@@ -12,24 +12,24 @@
...
     
-    Currently offering Bedwars and Skyblock, with more to come.">
+    Currently offering Derpitcraft, Bedwars and Skyblock, with more to come.">
 
     <meta property="twitter:url" content="https://trix.tk/">
     <meta property="twitter:title" content="TrixMC">
     <meta property="twitter:description" content="A free Minecraft server network unlike any other. 
-    Supports Minecraft Java and Bedrock 1.7.X to 1.17.X, with cracked client support.
+    Supports Minecraft Java and Bedrock 1.8.X to 1.17.X, with cracked client support.
```
Comment (vachanmn123) [inline index.html:29]: max version to be 1.18.1

**Review by vachanmn123 — COMMENTED** [review]
Please make these changes

**Review by PjrCodes — CHANGES_REQUESTED** [review]
see what @vachanmn123 said. Also add some text to the derpitcraft section saying REGISTRATIONS HAVE STARTED!!!

**Review by PjrCodes — APPROVED** [review]
can merge

**Pull request merged**